_IBC/archway-jackal.json

```diff
@@ -1,23 +1,23 @@
 {
-  "$schema": "../ibc_data_schema.json",
+  "$schema": "../ibc_data.schema.json",
   "chain_1": {
     "chain_name": "archway",
-    "client_id": "07-tendermint-5",
-    "connection_id": "connection-4"
+    "client_id": "07-tendermint-17",
+    "connection_id": "connection-18"
   },
   "chain_2": {
     "chain_name": "jackal",
-    "client_id": "07-tendermint-59",
-    "connection_id": "connection-49"
+    "client_id": "07-tendermint-61",
+    "connection_id": "connection-50"
   },
   "channels": [
     {
       "chain_1": {
-        "channel_id": "channel-3",
+        "channel_id": "channel-14",
```

Comment:
You can check this at Mintscan.
Working:
https://www.mintscan.io/archway/relayers/channel-14/not-established/channel-41

Not working / old one:
https://www.mintscan.io/archway/relayers/channel-3/not-established/channel-40

Reply:
Typically an IBC channel that's not working should be recovered rather than a new one created. It looks like there have already been tokens transferred on the old channel

Reply:
Yes, this is true.
This was calculated choice that was made by our partner (note that we do not operate these), the amount were minimal and not worth of the gov proposal since the partner needed working path fast.
As you can see the new/current channel is active with funds being transfered.

Reply:
I can see that the total amount transferred is a tiny tiny fraction on what the newer channel has, so I think this new channel with remain as canonical